wordpress-mobile/WordPress-iOS
# History / Revision: new CoreData model version

This PR introduces:
- `WordPressKit 1.4.3-beta.2` with the new *revisions* property mapped for `RemotePost`.
- New *CoreData* model version *82*
- New *revisions* property for `AbstractPost` entity.
- `AbstractPost` model has a new *revisions* property.
- New entity `Revision`:  it will contain the data of a *post* revision, like title, content, date etc...
- New entity `RevisionDiff`: it will contain the data for a *revision diff*, like the amount of additions or deletions and the revision id it refers to.
- New entities `RevisionDiffAbstractValue`, `RevisionDiffContentValue`, `RevisionDiffTitleValue`: these will contain the type of change and the operation type.


**To test:**
- Run the app over any branch with CoreData model version 81
- CoreData migration should work with no problem
- Select a wp.com blog and load the blogs list. Everything should work fine
